### DISCUSSION TITLE
[Feature Request] Allow always authorizing preview deployments from forks on a private repo

### DISCUSSION THREAD

**Discussion by cryptiklemur** [vercel/vercel#4214] [General]
Our company uses the fork flow for development (all developers have a fork, and create PRs against the upstream repo). 

If we want to take advantage of the environment variables or secrets, we have to authorize every single PR's preview deployment. This is very tedious.

Investigating the GitHub API, this isn't something that can be automated either.

**Comment by coetry** [comment]
Hi @aequasi this is a solid request. We're exploring what our options are internally to make this work.

**Reply by cryptiklemur** [reply]
Awesome to hear! Thanks :)
